solutions/landing-zone-namespaced/landing-zone/setters.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+  annotations:
+    internal.kpt.dev/upstream-identifier: '|ConfigMap|default|setters'
+    config.kubernetes.io/local-config: "true"
+data:
+  #############
+  # General Settings Values
+  #
+  # The following are Settings for the environment to bootstrap with
+  #
+  #############
+  billing-id: "0000000000"
+  org-id: "0000000000"
+  lz-folder-id: '0000000000'
+  #############
+  # Management Project
+  # This is the project where the config controller instance is running
+  # Values can be viewed in the Project Dashboard
+  management-project-id: management-project-12345 
+  management-project-number: "0000000000"
+  management-namespace: config-control
```

Comment:
Are any services planned on being deployed to the default `config-control` namespace? If not we could remove this value.

Reply:
my plan is that the config-control namespace (yakima SA) will be granted org admin privilege and will be used to manage IAM persmissions only. Down the road, I was thinking of creating another management namespace with the org admin privilege and use that one instead of the config-control namespace to avoid granting very high level permission to that yakima SA which comes from an external project

Reply:
Did you want to leave it as is for now and patch that in a future release?